public/class-lkn-payment-banking-slip-pix-for-lifterlms-public.php

```diff
@@ -86,6 +86,6 @@ public function enqueue_scripts(): void {
          * class.
          */
 
-        wp_enqueue_script( $this->plugin_name, plugin_dir_url( __FILE__ ) . 'js/lkn-payment-banking-slip-pix-for-lifterlms-public.js', array('jquery'), $this->version, false );
+        wp_enqueue_script( 'lkn-payment-banking-slip-pix-for-lifterlms', plugin_dir_url( __FILE__ ) . 'js/lkn-payment-banking-slip-pix-for-lifterlms-public.js', array('jquery'), $this->version, false );
```

Comment:
@FabricioLinkNacional Add the function '[localize_script](https://developer.wordpress.org/reference/functions/wp_localize_script/)' to load the string translations